python/src/LatentVariableModel_doc.i.in

```diff
@@ -0,0 +1,144 @@
+rely on a constant functional basis.
+
+See Also
+--------
+CovarianceModel
+
+Examples
+--------
+Create a latent model covariance function with a latent space of dimension 2, for a discrete variable 
+characterized by 3 levels:
+
+>>> import openturns.experimental as otexp
+>>> covModel = otexp.LatentVariableModel(3, 2)
+>>> activeCoordinates = [0.1, 0.3, -0.4]
+>>> covModel.setLatentVariables(activeCoordinates)
+>>> print(covModel(1, 2))
+[[ 0.904837 ]]
+>>> print(covModel(0, 2))
+[[ 0.882497 ]]
+>>> print(covModel(1, 1))
+[[ 1 ]]"
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::LatentVariableModel::computeAsScalar
+"Compute the covariance function for scalar model.
+
+Available usages:
+    computeAsScalar(s, t)
+
+Parameters
+----------
+s, t : floats, must have integer values between :math:`0` and :math:`l-1`
+
+Returns
+-------
+covariance : float
+    Covariance.
+
+Notes
+-----
+The method makes sense only if the dimension of the process is :math:`d=1`.
+It evaluates :math:`C(s, t)`."
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::LatentVariableModel::getActiveLatentVariables
+"Active latent variables accessor.
+
+Parameters
+----------
+activeLatentVariables : :class:`~openturns.Point`
+    Active coordinates of the discrete levels in the latent space. 
+    The inactive coordinates are set to 0 (i.e., the first latent 
+    variable is placed in the Euclidean space origin, and the second
+    latent variable can only be placec onto the first axis)."
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::LatentVariableModel::getFullLatentVariables
+"Latent variables accessor.
+
+Returns
+-------
+activeLatentVariables : :class:`~openturns.Point`
+    Coordinates of the discrete levels in the latent space."
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::LatentVariableModel::getLatentDimension
+"Latent dimension accessor.
+
+Returns
+-------
+LatentDimension : int
+    Dimension of the latent space."
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::LatentVariableModel::getNLevels
+"Number of levels accessor.
+
+Returns
+-------
+NLevels : int
+    Number of levels characterizing the discrete variable."
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::LatentVariableModel::setLatentVariables
+"Number of levels accessor.
+
+Parameters
+----------
+LatentVariables : :class:`~openturns.Point`
+    Active coordinates of the discrete levels in the latent space."
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::LatentVariableModel::draw
+"Not yet implemented"
```

Comment:
Is that necessary?

Reply:
I believe it is necessary, as the parent class provides a documented implementation of the draw method.